tutorials/online_analysis/lattice/online_analysis.ipynb

```diff
@@ -264,7 +266,7 @@
f57w5bKHS/P3V2UnxgFjlXzTAkxCNKUUj2/bfMS/0oiVVsQwdBFb8rrPuK//W+B//q/lv36i794+eD/E8erRsW/9+9h/Wf/TPIA/qv/Crd//59X151ucB9uALTfg/MurZCyXhFFOkEC0ErTAORjdNfL+4X5DkMbi75oib5qqAZTpBdE5BlIUy/zd3tcCsSD4Lw6ZPiWtOnC1sYxSuwsOwAX6Qz48OBCYxTpj2HFOHYyl5QsvtFoxV8477ICGuUiJQDHIw77Pc4P0QM6YGMqkUGjoENsBMekZaXFkbokc4xQ28XPQ2e/MoRdf7AG3b/Ulhp9TSnkmZG/EI20uGZ+R0MPTdFnKVaneZK/1bh4RcgAVXCpEKrnVJXEPANjks+aNzQXSzIfhtbNXDfwriJsqLor8Epvg80wwKfIGndTaJJXbzdg3IUGIbND1B/0fSv8kFH5uH7c0F0inEooUd32QN2nRMVl2Kvwkr28v6/z5v6aAlAico4Yh7XpWE5PiXXqnWcE1yeAygAA7ZZcB+mesoxG0zC40o6hFw8Pexk4YypGWe4aIjquuxT0YUGaYhsh4AAuhjJ4/t4inzXZnOES3eVsJbbpPSHQ5KuO2uSA6rF2TJ+0Ysqjyv3DeKsvmyYLcerD5gLphENoFf2tnKJnm1tu/CII+s/3+x4RnC6LdvRTsGdaNToeUGhj1k0GKxPa76suJHxNP+e9Zd66AHA4jIZjseussF/xetj7S2k8WCm598BtBnUqXVxfWuV9ux7gJX/y52b8Tz+7ph4FEfFQ1ckVXdUzcjLwNaSIOAwIukVr6tEdDhXh5yHyvjo7Yr+HeCfcPHmePj+wsAmj0zt3vHInGO3TY2oRBXD3oBSE5PQUx9RIyuSzN/TiSXJekaVEIFWvZRMmO0gyOIqb+1bRguiTnjzo9bLQzEuVya8y3c34Rco+AK0I01cFtBRYU5nBWVqlAKG6UGkQxHwRlFvRy2t8I/WF3w+NoJ+PcjF2O/dljVMfwyqlk1JCSDVu3QvYlKDo764maQ8dLnPEskTJ3s8Z3eWMfprw9NS1QsO9dgw3afyTJkOi5cjlEMyqpzGREkLZWGCHgykgW1Ts6kqxp9ShU1RinqVG9rK7lxhFvbv7vatjTNe3BSoq4uTi+UOoRhYuF6zvv8fzcw0LGZ2rHanG/lrMtsYZllI7IUrDnyhzLQWY7sSwwg6TokBRd4mKDD/KeUV9dgy1OyyTX2gwWigVFV5lCNfSY/zP//NXJM3/xQaNkP/mvwH+3/+8VnUgI1aYZcYOy9JhXUckvQZxvlRmGALiJAjk7aZuQY0fB4Ya0hTUbat3gXGYu6N6lh4fpRLNJ5kWz2IYesS9CBK60efSyc8pYFw1nkOcmzr7h7uqxIXQSbOuywXxeER8eMC5jFiHHp0CBT1frhPosCIyc9xZpaSLaYroyoLLqYZRGpJ2OuH7H36QL5QCoK6d+dX7fcSoEuvjR3mtecXyDUvpIY3GYLXPAWCviq7F9mtJk9twh+wrIB1PdUNVKy1aRYZN44jYssdBBSuj1Up/1kpSpuh//Gge1POlw3x6qejgkhvJOKMmdKcUEYLEDq8rMIZiYEKFlXsjSfKRDnCbdyfkpWEQvcYaYxjAnnFu2R4LESWRfF0RrjhNGIYaZ77eSYK1lDDsGkUeIbQVl2Z5CTs28xz58eM5YreLiKwQBKBTbw3XG4cFSFH5XKyep7Q2fPCWO1FuLheco/SrGB91/4ZgBqrclVG8NJ8+oX834EpLTIET2Q/NxXKoqBnZSerdm7v/iZDxW7NdUhKEnry4qcmPEaPru9GlhDDdybNYNUfBsqV0oqSEFTGqIeSs6BWdOfmAiqoug3h7Hx+FXvvLF/Qp4aTeZFOQvJXBsBdNgKSyb3bPMAgRfPxoXu9zGXH6CHz3kK2SmkcwaKg0oREatmVz1gndH7Y+14AICftknxE+uk9Cf/Q8TVPnltNZDDrX1mFF12lREOYCHsRifPNAfnGqhjpn4I1WWlTzjO50Qn844FR6K1LAMMG7O7TCUxn7upczPowZCGLok40KgBmbMJcQ4KqohaqzOCDRqwOkXeOBVKZSQkavAReyT4eDi3UPtQIYq56lJIbAWM64zjvMc8S9gpsET3wkNp3NMULAFwNqnbdwru6iglhzZbgQ8jBdIz2L1fiTTbEQO3dGFTmtZC33V4yV4yc1RAaYF/y87KwX0G6H6i1Xfmk5FnCVv1TRYrANvTdRdUZGPpg+4A0Ejxp/ZfzJyj6RRP7bt8Tm4dREXVkcO+xJSTgJYxju7tC9e2e7HXWtq2ZTE/31biNagTc1KsYBRoldKGCHtvNFmQATalKy/aDinXMbE8ZL+vhYSxTKo7u6JiRFzDT2U8MYnp/lUH1beTKjiEWzpCuDApxFrXHypci+8qJYd81S8PDQ3u8QHHNzIU92AKVImSvtIuujFmIKtuddvmG3682FJC78KF4FVMvS9h/VTf/4KK+3clGK5Ht0z7wGmwQUX5nIFm8jWqSPue0AgFn4+z3wr/4V8B//x0DOGPF/XojOv82wOf7Lf4n1X/5L+ff798B//98D//Sf2v4k3Rsy91JqTHTlTBsDyHNj99ltR2QAlU7mGWmqwtlFV4EdU4mq8DPxcKiKEm443CXggspQlfZiCDheyPhjjVnOGYN2xFyJKOOVkDCXhGWL9SMEvHlYLcQtpR79u3eV6LjglECPnFfqSavhVLeLITlxAqZJ5+YQsVi06ykFozITr9jaO91dXNEZunPYuQR4HeJ6dt4u3ey7u3qs69Cje3y03229Nna+Dw81tC0Icsln+Nd2ne6///IWrXcud7qO+BxDtvS75AteF+OoCX5uwtw/3dfdlDCo58TTLT/LkBV7rG46f24CP4uyTLBgnoGdnvU6jOjKUkPSbPMqimnJrseLCWWUIvxwlhf1SYEIp3AafYQgKHCBJRzGCEPk2UCa+yNyJAI08nI2b5fdaz0XGveba2DPWjVeX
```

Comment:
I have no idea what changed here. The line is insanely long with all the image data. Please make sure the image still loads and renders properly before merging.

Reply:
Yah not sure why this happened, but it looks ok